tests/unit/acl-v2.tcl

```diff
@@ -75,7 +77,8 @@ start_server {tags {"acl external:skip"}} {
 
     test {Test separate read permission} {
         r ACL SETUSER key-permission-R on nopass %R~read* +@all
-        $r2 auth key-permission-R password
+        catch {$r2 auth key-permission-R password} err
+        puts $err
```

Comment:
Was this leftover from debugging?

Reply:
Yeah, it was from debugging. Doesn't throw error.